content/desktop/hardened-desktop/settings-management/configure.md

```diff
@@ -186,6 +190,7 @@ The following `admin-settings.json` code and table provides an example of the re
 | `blockDockerLoad` | | If `value` is set to `true`, users are no longer able to run [`docker load`](/reference/cli/docker/image/load/) and receive an error if they try to.|
 | `filesharingAllowedDirectories` |  | Specify which paths your developers can add file shares to. Also accepts `$HOME`, `$TMP`, or `$TEMP` as `path` variables. When a path is added, its subdirectories are allowed. If `sharedByDefault` is set to `true`, that path will be added upon factory reset or when Docker Desktop first starts. |
 | `useVirtualizationFrameworkVirtioFS`|  macOS only | If `value` is set to `true`, VirtioFS is set as the file sharing mechanism. Note: If both `useVirtualizationFrameworkVirtioFS` and `useGrpcfuse` have `value` set to `true`, VirtioFS takes precedence. Likewise, if both `useVirtualizationFrameworkVirtioFS` and `useGrpcfuse` have `value` set to `false`, osxfs is set as the file sharing mechanism. |
+| `useVirtualizationFrameworkRosetta`|  macOS only | If `value` is set to `true`, Turns on Rosetta to accelerate x86/AMD64 binary emulation on Apple Silicon. Note: enabling this option will automatically enable `Use Virtualization framework`. |
```

Comment:
> Turns on Rosetta to accelerate x86_64/amd64 binary emulation on Apple Silicon. Note: You must have Virtualization framework enabled.

```suggestion
| `useVirtualizationFrameworkRosetta`|  macOS only | If `value` is set to `true`, Turns on Rosetta to accelerate x86/AMD64 binary emulation on Apple Silicon. Note: Enabling this option will automatically enable `Virtualization framework`. |
```
I'm not sure about the capital "E", but that's how it is in the dashboard. @aevesdocker?